pom.xml

```diff
@@ -122,7 +121,7 @@
 			<dependency>
 				<groupId>org.apache.camel</groupId>
 				<artifactId>camel-spring</artifactId>
-				<version>${camel-version}</version>
+				<version>${camel-community-version}</version>
```

Comment:
should we productize this dependency?

Reply:
We might want to - but it doesn't appear that it is necessary (I can compile everything in spring-boot directory without it).  Removed the dep.

Reply:
https://github.com/jboss-fuse/camel/pull/2142